lib/src/call_chain_visitor.dart

```diff
@@ -311,7 +311,8 @@ class CallChainVisitor {
 
     // Don't split right after a collection literal.
     if (expression is ListLiteral) return false;
-    if (expression is MapLiteral) return false;
+    // TODO(rnystrom): should we return false for sets as well?
```

Comment:
Yes, you can remove the `isSet` check here too. (When I added support for set literals, I didn't think to search for `MapLiteral` everywhere and see where else sets should get special handling.)